composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.0",
-        "google/auth": "^1.18.0",
+        "google/auth": "1.19.1||^1.25.0",
```

Comment:
in `google/auth:1.20.0`, we dropped support for PHP 7.0, which is still supported by this library. For this reason, we had to configure the dependency like this.